lib/retirement_schemes/classification_count.rb

```diff
@@ -4,8 +4,16 @@ def initialize(count)
       @count = count
     end
 
-    def retire?(count)
-      count.classifications_count >= @count
+    def retire?(sw_count)
+      if disabled?
```

Comment:
If you invert this to `enabled?`, this method body could just be `enabled? && sw_count.classifications_count >= @count` which I think reads better.